stack/keycloak/migration/add-user-attribute-mapper-wahlbezirks-art.yml

```diff
@@ -0,0 +1,12 @@
+id: add user attribute mapper wahlbezirksart
+author: MrSebastian
+realm: ${SSO_REALM}
+changes:
+  # maps user attribute user_name to a claim user_name
```

Comment:
This comment seems wrong?

Reply:
fixe with f57806c78f1588753ce69000fc6e61f111c9c52c

Reply:
fixed with afac4ae160b1e6e76fe581e14e438f75a1755a60